content/jobs.md

```diff
@@ -1,6 +1,89 @@
 ---
 title: Jobs
-summary: |
-  **We are not currently accepting applications for any positions** --
-  thanks for your interest!
+banner: Web Project Manager
+image:
+  svg: logos/oddbird
+  type: angle
+  attrs:
+    style: '--align: center'
+sub: Helping guide the web design and development process
 ---
+
+OddBird is looking for a Project Manager to join the team.
+
+## We’re Looking for Someone Who:
+
+- Has knowledge of the web design and development process
+- Can manage multiple projects simultaneously
+- Will help plan, monitor, and facilitate projects on time and within budget 
+- Is an excellent verbal and written communicator
+- Is not tied to a single methodology, 
+  but can help guide the team in finding & implementing 
+  processes that work for us
+- Is located in a US time-zone
```

Comment:
Technically this includes, like, Hawaii and American Samoa  and Guam :) Would it be both more accurate and less US-centric to say e.g. "is located in a continental North America timezone"?